client/landing/stepper/declarative-flow/hundred-year-domain.ts

```diff
@@ -87,7 +87,11 @@ const HundredYearDomainFlow: Flow = {
 			}
 		}
 
-		return { submit };
+		const exitFlow = ( location = '/sites' ) => {
```

Comment:
what does this change do?

Reply:
It creates the `exitFlow` function - so we have a way out of the flow using the frameworks provided tools - and returns it within the `useStepNavigation` hook of it.
We do it to handle this according to the Stepper framework - continue on the given path and quit the flow.